app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::SmartStateScan < ApplicationHelper::Button::Basic
+  needs :@record
+
+  SMART_ROLES = %w(SmartProxy SmartState).freeze
```

Comment:
@enoodle @roliveri should this constant live somewhere in the SmartState subsystem?